app/operators/dcm2nii_operator.py

```diff
@@ -40,11 +39,16 @@ def compute(self, op_input: InputContext, op_output: OutputContext, context: Exe
         for f in input_files:
             shutil.move(os.path.join(input_path, f), dcm_input_path)
 
+        # create output directory for input-ct-dataset.nii.gz
+        dcm_output_path = os.path.join(input_path, 'nii_ct_output')
+        if not os.path.exists(dcm_output_path):
+            os.makedirs(dcm_output_path)
+
         # Run dcm2niix
-        subprocess.run(["dcm2niix", "-z", "y", "-o", input_path, "-f", "input-ct-dataset", dcm_input_path])
+        # subprocess.run(["dcm2niix", "-z", "y", "-o", dcm_output_path, "-f", "input-ct-dataset", dcm_output_path])
```

Comment:
This should not be commented out for production.

Reply:
un-uncommented